pages/community-call.jsx

```diff
@@ -50,25 +50,25 @@ var CommunityCallPage = React.createClass({
           <h3>Workshop Video Stream</h3>
 
           <div className="video-wrapper">
-            <iframe width="560" height="315" src="//www.youtube.com/embed/d3YJUZVoYws" frameborder="0" allowfullscreen></iframe>
+            <iframe width="560" height="315" src="//www.youtube.com/embed/LlEXY8NKxoY" frameborder="0" allowfullscreen></iframe>
           </div>
 
           <h4>
             Open Agenda
-            <a title="Open the agenda in a new tab" className="fa fa-external-link open-etherpad" href="https://public.etherpad-mozilla.org/p/mozcommunitycallApril16">
+            <a title="Open the agenda in a new tab" className="fa fa-external-link open-etherpad" href="https://public.etherpad-mozilla.org/p/mozcommunitycallMay16">
             </a>
           </h4>
 
-          <iframe className="etherpad" src="https://public.etherpad-mozilla.org/p/mozcommunitycallApril16"></iframe>
+          <iframe className="etherpad" src="https://public.etherpad-mozilla.org/p/mozcommunitycallMay16"></iframe>
 
           <h2>Upcoming Calls</h2>
 
           <ul className="upcoming-workshops">
             <li>
-              <p className="date">Wednesday, May 25 - Time TBD</p>
-              <h2>Theme: Gigabit</h2>
+              <p className="date">June 29th</p>
+              <h2>Theme: Summer Learning</h2>
               <p>
-                Featured Speakers: TBD
+                 Featured Speakers: TBD
```

Comment:
In general, try to keep the whitespace (indentation, etc) the same when not changing content.